mrmustard/lab_dev/states/base.py

```diff
@@ -65,6 +65,7 @@ def __rshift__(self, other: CircuitComponent) -> CircuitComponent:
             dm = DM()
```

Comment:
I don't know why we're not using `_from_attributes` here (I probably forgot to update this), but I believe that `dm = DM._from_attributes("", component.representation, component.wires)` should do the job of these 4 lines